Examples/Algorithms/Vertexing/src/TruthVertexSeeder.cpp

```diff
@@ -0,0 +1,59 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "TruthVertexSeeder.hpp"
+
+#include "VertexingHelpers.hpp"
+
+namespace ActsExamples {
+
+TruthVertexSeeder::TruthVertexSeeder(const Config &cfg) : m_cfg(cfg) {}
+
+Acts::Result<std::vector<Acts::Vertex>> TruthVertexSeeder::find(
+    const std::vector<Acts::InputTrack> & /*trackVector*/,
+    const Acts::VertexingOptions &vertexingOptions,
+    Acts::IVertexFinder::State &anyState) const {
+  auto &state = anyState.template as<State>();
+
+  if (state.nextVertexIndex >= m_cfg.vertices.size()) {
+    return std::vector<Acts::Vertex>();
+  }
+
+  const auto &truthVertex = m_cfg.vertices[state.nextVertexIndex];
+  ++state.nextVertexIndex;
+
+  Acts::Vertex converted;
```

Comment:
This is effectively the seed right?

Reply:
Right this can use a better name.

At some point I wanted to decouple the seeds from the vertex EDM since they only share a few properties similar to vertex seeder and vertex finder.